docs/en/wallet-provider-endpoint.rst

```diff
@@ -446,26 +442,314 @@ The ``nameSpaces`` for the domestic nameSpace Json Objects are defined as follow
+      - [:rfc:`7516#section-4.1.1`]
+    * - **kid**
+      - Thumbprint of the Wallet Instance's JWK contained in the ``cnf`` claim.
+      - [:rfc:`7638#section_3`]
+    * - **typ**
+      - The type of the JWT, it MUST set to ``wp-wua+jwt``.
+      -
+
+The Wallet Unit Attestation Request JWT includes the following body claims:
+
+.. list-table::
+    :class: longtable
+    :widths: 20 60 20
+    :header-rows: 1
+
+    * - **Claim**
+      - **Description**
+      - **Reference**
+    * - **iss**
+      - The identifier of the Wallet Provider concatenated with the thumbprint of the JWK in the ``cnf`` claim.
+      - [:rfc:`9126`], [:rfc:`7519`].
+    * - **aud**
+      - The identifier of the Wallet Provider.
+      - [:rfc:`9126`], [:rfc:`7519`].
+    * - **exp**
+      - UNIX timestamp representing the JWT expiration time.
+      - [:rfc:`9126`], [:rfc:`7519`].
+    * - **iat**
+      - UNIX timestamp representing the JWT issuance time.
+      - [:rfc:`9126`], [:rfc:`7519`].
+    * - **nonce**
+      - The ``nonce`` obtained from the Nonce Endpoint.
+      -
+    * - **hardware_signature**
+      - The signature of ``client_data`` obtained using the Cryptographic Hardware Key, encoded in the ``base64url`` format.
+      -
+    * - **integrity_assertion**
+      - The Integrity Assertion obtained from the **Device Integrity Service APIs** with the holder binding of ``client_data``.
+      -
+    * - **key_attestation**
+      - The key Attestation obtained from the **Key Attestation APIs** with the holder binding of ``client_data``.
+      -
+    * - **hardware_key_tag**
+      - The value of the Cryptographic Hardware Key Tag.
+      -
+    * - **cnf**
+      - JSON object containing the public part of an asymmetric key pair owned by the Wallet Instance.
+      - :rfc:`7800`.
+
+Below is a non-normative example of a Wallet Unit Attestation Request JWT header and payload.
+
+.. note::
+  The Wallet Unit Attestation Request JWT body claims based on the OS of the user's device contain either **integrity_assertion** (in case of iOS) or **key_attestation** (in case of Android).
+
+.. code-block:: json
+
+    {
+      "alg": "ES256",
+      "kid": "OnsiandrIjp7ImNydiI6IlAtMjU2Iiwia3R5IjoiRUMiL",
+      "typ": "wp-wua+jwt"
+    }
+
+.. code-block:: json
+  
+    {
+      "iss": "https://wallet-provider.example.org/instance/OnsiandrIjp7ImNydiI6IlAtMjU2Iiwia3R5IjoiRUMiL",
+      "aud": "https://wallet-provider.example.org/",
+      "nonce": "f3b29a81-45c7-4d12-b8b5-e1f6c9327aef",
+      "hardware_signature": "KoZIhvcNAQcCoIAwgAIB...",
+      "integrity_assertion": "o2NmbXRvYXBwbGUtYXBwYXNzZXJ0aW9uLXBheWxvYWQtYXBw...",
+      "hardware_key_tag": "QW12DylRTmF89iGkpydNDWW7m8bVpa2Fn9KBeXGYtfX"
+      "cnf": {
+        "jwk": {
+          "crv": "P-256",
+          "kty": "EC",
+          "x": "8FJtI-yr3pjyRKGMnz4WmdnQD_uJSq4R95Nj98b44",
+          "y": "MKZnSB39vFJhYgS3k7jXE4r3-CoGFQwZtPBIRqpNlrg"
+        }
+      }
+    }
+
+
```

Comment:
This seems like a repetition of what's already stated [here](https://github.com/italia/eid-wallet-it-docs/blob/da0574c4fbdde76fd647e4df59a84c7b83c87583/docs/en/mobile-application-instance.rst?plain=1#L282), which is cited above for the WAA. I believe an inconsistency has arisen that I don't see addressed in this PR. Furthermore, it's unclear what the endpoint is from the regulatory text.